src/ZendDeveloperTools/EventLogging/EventContextProvider.php

```diff
@@ -66,7 +66,19 @@ public function getEvent()
     public function getEventTarget()
     {
         $event = $this->getEvent();
-        return (is_object($event->getTarget())) ? get_class($event->getTarget()) : (string) $event->getTarget();
+        $target = $event->getTarget();
+
+        if (is_object($target)) {
+            return get_class($target);
+        } elseif (is_array($target)) {
```

Comment:
Please don't use `elseif`. Since you are returning early, you don't even need an `else` clause